components/calendar/test/calendar.visual-diff.js

```diff
@@ -60,8 +60,12 @@ describe('d2l-calendar', () => {
 				await page.evaluate((lang, firstCalendarOfPage) => {
 					const calendar = document.querySelector(firstCalendarOfPage);
 					return new Promise((resolve) => {
-						calendar.addEventListener('d2l-localize-resources-change', resolve, { once: true });
-						document.querySelector('html').setAttribute('lang', lang);
+						if (document.querySelector('html').getAttribute('lang') !== lang) { // Needed for retries
+							calendar.addEventListener('d2l-localize-resources-change', resolve, { once: true });
+							document.querySelector('html').setAttribute('lang', lang);
+						} else {
+							resolve();
+						}
```

Comment:
I needed to add this fix a couple of places.  On retries, this would just timeout after 10 seconds